src/utils/src/dataset-utils.ts

```diff
@@ -388,6 +388,27 @@ export function getSampleForTypeAnalyze({
   return sample;
 }
 
+/**
+ * Check if string is a valid Well-known binary (WKB) in HEX format
+ * https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry
+ *
+ * @param str input string
+ * @returns true if string is a valid WKB in HEX format
+ */
+export function isHexWkb(str: string | null): boolean {
```

Comment:
This is interesting. I wonder how to handle HexWKB in loaders.gl. Maybe just add `HexWKBLoader` / `HexWKBWriter`?

Reply:
HexWKBLoader/Writer sounds great 👍